### ISSUE TITLE
Replace "comments" field with study log

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#688]
**TL;DR**: We should remove the comments field from the study table.

**Narrative**
Comments on a study are recorded in the `comments` field on the `Study` table and in the `extra` JSON object on the `Studylogs` table.  This redundancy can be eliminated by querying the `Studylogs` table for the comments and removing the comments field from study.

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests
- [ ] Scoping
- [ ] Alternatives Considered
- [ ] Rico sign-off

**Implementation Notes**
At the time of the creation of this issue, it is not known how extensively the comments on the Study table is being used.  A bit of research maybe needed to make sure this change doesn't get out of hand.
